src/common.ts

```diff
@@ -20,16 +20,21 @@ export class GaxiosError<T = any> extends Error {
   code?: string;
```

Comment:
Suggesting some doc for usage/clarity.

```suggestion
  /**
  * An Error code.
  * See {@link https://nodejs.org/api/errors.html#errorcode error.code}
  *
  * @example
  * 'ECONNRESET'
  */
  code?: string;
```